src/Blog/Repository/Posts/MySQL.php

```diff
@@ -34,7 +34,11 @@ public function getList(int $page = 1, int $postsPerPage = 10): array
 
     public function getOne(int $id): ?Model\PostView
     {
-        return empty($row) ? null : $this->createModel($row);
+        $sql = "select from Posts";
+        $stmt = $this->pdo->query($sql);
+        $data = $stmt->fetchOne();
+
+        return $data;
```

Comment:
Funkcja musi zwrócić obiekt klasy Model\PostView albo null.
Dlatego lepiej to zrobić tak:
`return empty($data) ? null : $this->createModel($data);`